### PR TITLE
Add rake task to grab the patch for a PR

### PR DESCRIPTION
**Pull request by i2amsam**
```
samp@tw-mbp-samp /crashlytics/fastlane/fastlane (master *) $ rake try_out[3763]
Getting pull request #3763:Updated cp command flag
```

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -167,6 +168,22 @@ task :features do
   end
 end
 
+desc 'show repos with checked-out feature-branches'
```
Comment (KrauseFx) [inline Rakefile]: Copied description?

File: Rakefile [hunk 2]
```diff
@@ -167,6 +168,22 @@ task :features do
...
+task :try_out, [:pull] do |t, args|
+  pr_num = args[:pull]
+  require 'faraday'
+  require 'faraday_middleware'
+  conn = Faraday.new do |req|
+    req.headers['Authorization'] = "token #{GITHUB_TOKEN}"
```
Comment (KrauseFx) [inline Rakefile]: Where does `GITHUB_TOKEN` come from?

File: Rakefile (hunk 1 above)
Comment (mfurtak) [inline Rakefile]: This description seems a little strange. Something like...
```
'Apply the code from the specified PR to your local working copy. Run with `rake try_out[PR#]`'
```
?

**Comment by KrauseFx** [comment]
I like it, looks great, I added some comments

**Comment by mfurtak** [comment]
This seems nice and simple and good. Things that it does not do include:

* Give you a git remote for the contributor's fork (maybe you don't care?)
* Give you a branch for your changes

Those things may not be desirable, but are things that I do for myself personally with a similar script, so I figured I'd mention it

File: Rakefile (hunk 1 above)
Comment (i2amsam) [inline Rakefile]: updated!

**Comment by i2amsam** [comment]
@mfurtak Updated.  I wasn't sure the pros and cons for setting the upstream instead.

File: Rakefile [hunk 3]
```diff
@@ -1,5 +1,7 @@
+require 'pry'
```
Comment (mfurtak) [inline Rakefile]: Do we want to leave `pry` required even when we're not debugging?

**Comment by mfurtak** [comment]
Looks good to me, with one question.

**Comment by mfurtak** [comment]
@i2amsam Still want to get this in?

**Comment by i2amsam** [comment]
Closing, this is probably not the best way to handle this.

**Pull request closed without merging**